docs/backendFlow.md

```diff
@@ -0,0 +1,243 @@
+# **Backend Flow 🕸️**
+
+<br>
+
+> ## Explaining the flow using `Users` created in the project ✌️
+
+<br>
+
+- [**Backend Flow 🕸️**](#backend-flow-️)
+  - [Controllers](#controllers)
+  - [Models](#models)
+  - [Base/ Global Route](#base-global-route)
+  - [Different HTTP Routes](#different-http-routes)
+  - [Middlewares](#middlewares)
+  - [config.js](#configjs)
+  - [server.js](#serverjs)
+  - [test](#test)
+  - [package.json file](#packagejson-file)
+    - [Want to Contribute ??](#want-to-contribute-)
```

Comment:
The order which I feel might help people understand would be:
- server.js (since this is the entry point)
- routes (after server.js, they should comes to routes)
- middleware
- controller
- model

Let me know what you think 😉 
These are just my views

Reply:
I was thinking about this too, but was not sure what my initial thought was.
BTW the suggestion is good 👌🏼